src/client/Wrapper.js

```diff
@@ -38,6 +39,7 @@ export default class Wrapper extends React.PureComponent {
     route: PropTypes.shape().isRequired,
     user: PropTypes.shape().isRequired,
     locale: PropTypes.string.isRequired,
+    username: PropTypes.string,
```

Comment:
I think that would be nicer to put it below required prop types.

Reply:
will do!